main/src/ca/uwaterloo/flix/language/phase/jvm/GenMainClass.scala

```diff
@@ -46,7 +92,9 @@ object GenMainClass {
     root.defs.get(sym) flatMap {
       case defn =>
         // The main function must take zero arguments.
```

Comment:
Not sure what is happening here.

Reply:
I also am not sure if this method is required since it was there already I just left it.